lib/search_files.js

```diff
@@ -0,0 +1,253 @@
+var fs = require('fs');
+var path = require('path');
+var oshomedir = require('os-homedir');
+
+var isFunction = require('lodash.isfunction');
+var isPlainObject = require('lodash.isplainobject');
+var isArray = require('lodash.isarray');
+var isString = require('lodash.isstring');
+var doNothing = require('lodash.identity');
+var mapValues = require('lodash.mapvalues');
+
+var isAbsolutePath = require('is-absolute');
+var parsePath = require('parse-filepath');
+
+function isWindows() {
```

Comment:
this can't change, so it should probably just be a variable defined here `var isWindows = process.platform === 'win32'`